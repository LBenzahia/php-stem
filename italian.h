
/* This file was generated automatically by the Snowball to ANSI C compiler */

extern struct SN_env * italian_create_env(void);
extern void italian_close_env(struct SN_env * z);

extern int italian_stem(struct SN_env * z);

