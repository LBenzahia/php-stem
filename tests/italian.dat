a
à
aa
aalst
ab
abakoumova
abano
abate
abati
abbacinare
abbacinati
abbadia
abbado
abbagliaron
abbagliato
abbaia
abbaiano
abbaiar
abbaini
abbandona
abbandonando
abbandonano
abbandonar
abbandonare
abbandonarla
abbandonarlo
abbandonarsi
abbandonarvi
abbandonasse
abbandonata
abbandonate
abbandonati
abbandonato
abbandonava
abbandonerà
abbandoneranno
abbandonerò
abbandono
abbandonò
abbaruffato
abbassamento
abbassando
abbassandola
abbassandole
abbassar
abbassare
abbassarono
abbassarsi
abbassassero
abbassato
abbassava
abbassi
abbassò
abbastanza
abbatté
abbattendo
abbattere
abbattersi
abbattesse
abbatteva
abbattevamo
abbattevano
abbattimento
abbattuta
abbattuti
abbattuto
abbellita
abbenché
abbi
abbia
abbiam
abbiamo
abbian
abbiano
abbiate
abbiccì
abbietti
abbigliamento
abbinamento
abbinano
abbinata
abbinato
abboccamenti
abboccamento
abbominata
abbominazione
abbominevole
abbominevoli
abbominio
abbonati
abbonato
abbonda
abbondano
abbondante
abbondantemente
abbondanti
abbondanza
abbondare
abbondio
abbonire
abbordarne
abbordo
abborracciata
abborrimento
abborrito
abbott
abbozzata
abbozzato
abbracci
abbraccia
abbracciamenti
abbracciar
abbracciare
abbracciarne
abbracciaron
abbracciata
abbracciate
abbracciati
abbracciato
abbracciavano
abbraccio
abbracciò
abbreviarli
abbreviata
abbreviati
abbronzate
abbronzati
abbruciacchiato
abbruciacchiavano
abbrunati
abbssanda
abburattavan
abc
abdel
abdic
abdica
abdul
abeille
abel
aberdeen
abergele
abete
abietto
abile
abili
abilità
abilitante
abilitate
abilitazione
abiliterà
abisso
abida
abidacolo
abidante
abitanti
abitare
abitada
abidate
abitati
abitato
abitator
abitatori
abitava
abitavano
abitazione
abitazioni
abiti
abito
abituale
abituali
abitualmente
abituarci
abituato
abitudine
abitudini
abituro
ablandarlos
abolendo
abolire
abolirli
abolisce
abolida
abolizione
aborto
abruzzesi
abruzzo
abulia
abusare
abuso
ac
accada
accadde
accaddero
accade
accademia
accademica
accademicamente
accademici
accademico
accademie
accadendo
accader
accaderà
accadere
accadesse
accadeva
accadevan
accadon
accadono
accaduta
accadute
accaduto
accalappino
accame
accampamento
accampati
accando
accanimento
accanite
accanito
accanto
accantonamenti
accantonamento
accantonando
accarezza
accarezzando
accarezzandolo
accarezzar
accarezzare
accarezzarla
accarezzarlo
accarezzato
accarezzavan
accarezzò
accartocciava
accasato
accatastata
accattabrighe
accattando
accattar
accattarlo
accattato
accattivanti
accatto
accattoni
accavallata
acccordi
accecato
accedatis
accedere
accelera
accelerando
accelerare
accelerato
acceleratore
accelerazione
accellerare
accenda
accende
accendere
accenna
accennando
accennano
accennar
accennare
accennarla
accennarono
accennate
accennati
accennato
accennava
accennavano
accenneremo
accennerò
accenno
accennò
accenti
accento
accentrato
accentua
accentuandone
accentuano
accerchiati
accertamenti
accertamento
accertare
accertarmi
accertarsi
accertate
accertato
accertò
accesa
accese
accesi
acceso
accessi
accessibili
accesso
accessòri
accessorie
accetta
accettabile
accettabili
accettando
accettano
accettare
accettarle
accettaron
accettarsi
accettasse
accettata
accettate
accettati
accettato
accettava
accettazione
accette
accetterà
accetteresti
accetterò
accetti
accetto
accettò
accezione
acchetare
acchiappar
acchiappare
acchiapparlo
acchiappato
acchiappatolo
acciai
acciarino
accidentale
accidentalmente
accidente
accidenti
accieca
accigliato
accingo
acciò
acciocché
accipigliato
accissan
accisghe
acclamazione
acclamazioni
accluder
accluderla
accoglie
accogliendo
accoglienza
accoglienze
accogliere
accoglierli
accogliervi
accoglieva
accoglimento
accolse
accolta
accoltellamento
accoltellarlo
accoltellata
accolti
accolto
accoatotò
accomatotò
accomoda
accomodamento
accomodando
accomodandosi
accomodano
accomodar
accomodarcisi
accomodare
accomodarla
accomodarmi
accomodarsi
accomodata
accomodate
accomodati
accomodato
accomodava
accomodavano
accomoderemo
accomoderò
accomodo
accomodò
accompagna
accompagnamento
accompagnando
accompagnandolo
accompagnano
accompagnar
accompagnare
accompagnarla
accompagnarlo
accompagnaron
accompagnata
accompagnate
accompagnati
accompagnato
accompagnatolo
accompagnava
accompagnavano
accompagnerete
accompagnerò
accompagni
accompagnò
acconcio
acconsente
acconsentì
acconsentire
acconsentirò
accontentarsi
accoppia
accoppiamento
accoppiata
accora
accoramento
accorando
accorata
accorciato
accorda
accordarlo
accordarsi
accordasse
accordate
accordato
accordava
accorderebbero
accordi
accordo
accorga
accorge
accorgendosi
accorgerà
accorgere
accorgermi
accorgersene
accorgersi
accorgesse
accorgeva
accorgevan
accorgo
accorgono
accorpa
accorpamento
accorpando
accorre
accorrere
accorresse
accorreva
accorrevano
accorrono
accorse
accorsero
accorso
accortamente
accorti
accorto
accortosi
accosta
accostamento
accostandosi
accostare
accostarglisi
accostarsele
accostarsi
accostate
accostatevi
accostati
accostato
accostatosi
accostava
accosto
accostò
accostumato
accovacciati
accozzaglia
accozzando
accozzar
accozzarsi
accozzati
accrebbe
accreditamento
accreditati
accrediti
accresce
accrescendo
accrescer
accrescere
accrescerebbe
accrescerle
accrescerlo
accresceva
accrescevano
accresciuta
accucciata
accudire
accumulando
accumulato
accurata
accusa
accusando
accusandole
accusandolo
accusar
accusare
accusata
accusate
accusati
accusato
accusatore
accusava
accusavan
accusavano
accuse
accusò
acerba
acerbi
acerra
aceti
aceto
acetoselle
acevedo
achille
achillini
acidi
acireale
acklin
acli
acnur
acogliere
acopi
acoteias
acq
acqua
acquarone
acquatico
acquattato
acque
acquea
acquedotti
acquerugiola
acqui
acquieta
acquietamento
acquietando
acquietandosele
acquietante
acquietare
acquietarla
acquietarle
acquietarlo
acquietarono
acquietasse
acquietati
acquietava
acquietò
acquirente
acquirenti
acquisire
acquisirlo
acquisite
acquisito
acquisizione
acquisizioni
acquista
acquistandosi
acquistar
acquistare
acquistarne
acquistarsi
acquistata
acquistato
acquistavano
acquisterà
acquisteranno
acquisti
acquisto
acquistò
acquitrinosa
acquolina
act
action
acuta
acutamente
acute
acuti
acuto
ad
adagino
adagio
adam
adamo
adani
adatta
adadaginr