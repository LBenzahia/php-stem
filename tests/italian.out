a
Ã 
aa
aalst
ab
abakoumov
aban
abat
abat
abbacin
abbacin
abbad
abbad
abbagliaron
abbagl
abbai
abbai
abbai
abbain
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abbandon
abband
abbandon
abbaruff
abbass
abbass
abbass
abbass
abbass
abbass
abbass
abbass
abbass
abbass
abbass
abbass
abbass
abbast
abbatt
abbatt
abbatt
abbatt
abbattess
abbatt
abbatt
abbatt
abbatt
abbatt
abbatt
abbatt
abbell
abbenc
abbi
abbi
abbiamabbiamabbiamabbiamadgv